charts/rancher-monitoring/v0.1.0/questions.yml

```diff
@@ -1 +1 @@
-rancher_min_version: 2.4.0-rc1
+rancher_min_version: 2.3.7-rc1
```

Comment:
@loganhz can we make a PR into `release-v2.4` with this change?

Reply:
It’s already in release 2.4. https://github.com/rancher/system-charts/blob/release-v2.4/charts/rancher-monitoring/v0.1.0/questions.yml